plugins/westernsupport/spellpredictworker.cpp

```diff
@@ -88,10 +88,28 @@ void SpellPredictWorker::setLanguage(QString locale, QString pluginPath)
 {
     // locale for secondary layouts I.E., dvorak will be formatted as locale@layout
     // in this case we want to drop the layout portion
```

Comment:
It appears that this comment has become out of date now that this code handles `-` as much as it handles `@`.

Reply:
humm and i can't read easily my code, need to comment and/or change variable names